src/content/time-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+
+
+## How should I track?
+
+### Project’s tasks
+It was intentionally described „Harvest app to store tracked time”, because it’s not always a place where you should add tasks.
+**To track any project’s task we are using a JIRA-Harvest connection**, which allows you to start and stop timer related to current task without your special involvement.
+The only thing you should do is to find a „Harvest Time Tracking by 42nd” in a task on JIRA and then connect your harvest to it. If you did it, you can just start timer directly from a task card and **it will be automatically visible in your harvest timesheet**, with proper project, task category and description with a link to the task added.
+The only thing you should do is to click big green button in JIRA :) What a time saving!
+
+Warning: please **don’t track project tasks without JIRA usage**. It’s really important to do it this way because of not just easy linking. JIRA should has updated timers in tasks to verify estimated time vs spend time. It’s possible only this way.
+
+### Project’s stuff
+Each project has a list of categories in Harvest, but only a development (or similar) is what you can track by JIRA connection. **All project things, which are not tasks you should track by Harvest, choosing the related category (meeting, code review, estimation etc.) and adding a description.** The last thing is really important to always be sure what meeting/estimation took how much time. Descripition of code review time is not obligatory.
+
+OMG, the expected project is not on the list in Harvest, what should I do?
+If you want to do a code review (or something else) for somebody from not-your project, please ask your PM to enable this project for you. It’s important to ask before starting doing this, because your PM should be informed about it and finally decide is it OK.
+
+### Other tasks
+Luckily our working day is not only a project stuff. Each developer is added to „Dev Team Internal” project, which is created to track your time spent on developing monthly dev team tasks, learning, meetings (including 1on1s). Remember about a description to this kind of tasks, to make it clear what a dev task you are doing, what do you do while learning, what a meeting you’re participating etc.
+
+
+## Tracking good manners
+
+### Adding a description
+As mentioned a few times earlier, it’s an expected practice to add descriptions to your tracked positions. **Remember that you don’t need to describe them in details. Try to use a short definition of described task.** E.g. daily, planning, refinement (for meetings), Accessibility/A11y, TypeScript generics, Unit tests (for learning), a copy-pasted name of your dev task. The main goal of a description is to easy&fast define what it is, to easy&fast inform a PM about it.
```

Comment:
```suggestion
As mentioned few times earlier, it’s expected of you to add descriptions to your tracked positions. **Remember that you don’t need to describe them in details. Just add a short info, **e.g. daily, planning, refinement (for meetings), Accessibility/A11y, TypeScript generics, Unit tests (for learning), a copy-pasted name of your dev task. The main goal of a description is to provide an easily accessible info for the PM.
```

Reply:
IMO `a` in the first line should not be removed 👉 https://www.diki.pl/slownik-angielskiego?q=a+few+times

Reply:
🤔